truffle-artifact-loader.js

```diff
@@ -0,0 +1,15 @@
+const loaderUtils = require('loader-utils')
+
+module.exports = function (source) {
+    this.cacheable && this.cacheable()
+    let value = typeof source === 'string' ? JSON.parse(source) : source
+    const options = loaderUtils.getOptions(this)
+    let excludedKeys = options.exclude
+    if (excludedKeys != null) {
+        excludedKeys = excludedKeys.split(',')
+        for (let key of excludedKeys) {
+            delete value[key]
+        }
+    }
+    return value
+}
```

Comment:
Yeah. Mainly, I wrote it to filter out the bytecode from the build, but it could potentially be used for other things.